common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.identity.common.internal.broker;
+
+import android.app.Activity;
+import android.content.Intent;
+
+import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
+import com.microsoft.identity.common.exception.ClientException;
+import com.microsoft.identity.common.exception.ErrorStrings;
+import com.microsoft.identity.common.internal.logging.Logger;
+
+
+public class BrokerTokenProvider {
```

Comment:
Can we add some unit tests for this class?

Reply:
currently, there's only one method returning void, so it's not necessary to add UT for it now. When I add the implementation of "acquireTokenSilently", I will add UT for that method.